### ISSUE TITLE
[Feature Request] Social music, users listening activity and history, follow playlists, temp share with link

### ISSUE THREAD

**Issue by Extarys** [navidrome/navidrome#390]
There could be a User section on the sidebar with the list of users on the server, what they are listening to and their public playlists. We can then follow (easily access) that playlist in the future. As a server admin, I could see which artist is more popular and buy more of similar songs/albums, etc and manually delete songs we just didn't listen/didn't care for or something. This would make listening to music more social on a self-hosted platform :D

Most of my friends complaints about self hosted is the lack of social features. I agree to some extend that this (or some of it) would be awesome :D

For legal purposes, a user is a household member.

- [x]  Ability to share a song, an album or an artist using a link to a guest, with option for expiration date and transcoding (ie: expires in 7 days, transcode to mp3 128kbps) - **Not sure if copyright laws would allow that**, but I find myself sending a mp3 to my friend from time to time so we can discus the music style and he decide if he buy the album or not. (I don't use Youtube much) [Implemented with [pull/#2106](https://github.com/navidrome/navidrome/pull/2106)]
- [ ]  Ability to follow another user's playlist - or - star/fav a public playlist for easy access/put it at the top of shared/public playlists
- [ ]  See what other users are listening to
- [ ]  Listen along - ability to listen to the same thing as another user (boyfriend/wife/etc)
- [ ]  See listening history
- [ ]  See other users listening history
- [ ]  Follow an artist so we get a notification when more songs of that artist are added
- [ ]  Wishlist, users would be allowed to add a Discogs link for example to an artist or an album they like
- [ ]  Most played songs/artists across users on the server ( @mattbk )

**Comment by deluan** [comment]
Sorry for the late reply. Yes I'm planning to add most of this features in the upcoming releases. Will provide more details here when I start to work on these.

**Comment by mattbk** [comment]
As part of this, some sort of global "most played" list across users would be interesting.

**Comment by charliemartinez** [comment]
Good afternoon:
First of all I want to congratulate you on this application, I already have it running on my server and I am really surprised at how good it is.

I am very interested in the function to share music without logging in.

All the best
:-)
Charlie Martinez

**Comment by deluan** [comment]
@charliemartinez, thanks! Sharing without login (similar to Airsonic/Subsonic feature) is being tracked by #748

**Comment by griffin-rickle** [comment]
Bumping for "listen along" capability.

**Comment by Extarys** [comment]
Bad bot

**Comment by foinf** [comment]
> Bad bot

This is just the beginning of the real-life Terminator timeline.

**Comment by deluan** [comment]
I'll keep this open for now, but whenever I have the time will break it down in separated issues covering what aligns with Navidrome's goals.

**Comment by ChristinMilloy** [comment]
@deluan Appreciate you

**Comment by zeezhao** [comment]
Still a great idea. Thanks!

**Comment by raphmim** [comment]
>  Ability to follow another user's playlist - or - star/fav a public playlist for easy access/put it at the top of shared/public playlists

This is the only additional social feature I would to see 😇

**Comment by JoeJoeTV** [comment]
This is still relevant, especially sharing of playlists in my opinion.

**Comment by raphmim** [comment]
Agreed that following another user's playlists would be a great add!

**Comment by anshuman-d3** [comment]
Bumping for "listen along" capability. (2)

**Comment by nado** [comment]
Not sure what "listen along" really mean here, I’d be interested in something similar to what spotify offers:
Being able to share a common playing queue between multiple users with the sound being output from a single client device (making it different from the jukebox mode with the sound being output on the server)

**Comment by cmargiotta** [comment]
Bumping!

**Comment by anshuman-x64** [comment]
> Not sure what "listen along" really mean here, I’d be interested in something similar to what spotify offers: Being able to share a common playing queue between multiple users with the sound being output from a single client device (making it different from the jukebox mode with the sound being output on the server)

It is similar to Spotify Jam.
It's not exactly the playlist share feature. Rather a synced playback of the same song at same time, where both sides have control over the track and playback control.

**Comment by sakkarose** [comment]
Annual bump. This feature will always be needed as Spotify getting worse day by day.

**Comment by Gishky** [comment]
bumping "listen along" (spotify jam-like)... its like the biggest downgrade I experienced from stepping away from spotify
(ideally could also create a jam-link for guests so they can listen along as well but without adding stuff to the queue)

**Comment by deluan** [comment]
Spotify Jam-like is planned for the new UI/API, but I don't have a timeline yet.

**Comment by radiohertz** [comment]
@deluan are you open to contributions for this feature?
